azure/Guidelines.md

```diff
@@ -747,18 +747,70 @@ The ability to retry failed requests for which a client never received a respons
-while the Microsoft guidelines use the name "Operation-Location".
+:white_check_mark: **DO** implement an operation as an LRO if the 99th percentile response time is greater than 1s.
 
-:white_check_mark: **DO** support both `Azure-AsyncOperation` and `Operation-Location` HEADERS, even though they are redundant so that existing SDKs and clients will continue to operate.
+There are two basic patterns that can be used for long-running operations:
+1. Resource-based long-running operations (RELO)
+2. Stepwise long-running operations
+
+:white_check_mark: **DO** use the RELO pattern when the operation is on a resource that contains a "status" property that can be used to obtain the outcome of the operation.
+
+:ballot_box_with_check: **YOU SHOULD** only use the stepwise LRO pattern when the RELO pattern is not applicable.
+
+#### Resource-based long-running operations
+
+Some common situations where the RELO pattern should be used:
+1. A "create" operation for a resource where the basic structure of the resource is created immediately and includes a status field that indicates when the create has completed, e.g. "provisioning" -> "active".
+2. An action operation for a resource where both the initiation of the action and the completion of the action cause a change to the "status" property of the resource.
+
+:white_check_mark: **DO** return a `200-OK` response, `201-Created` for create operations, from the request that initiates the operation.  The response body should contain a representation of the resource that clearly indicates that the operation has been accepted or started.
+
+:white_check_mark: **DO** support a get method on the resource that returns a representation of the resource including the status field that indicates when the operation has completed.
+
+:white_check_mark: **DO** define the "status" field of the resource as an enum with all the values it may contain including the "terminal" values "Succeeded", "Failed", and "Canceled".
+
+:ballot_box_with_check: **YOU SHOULD** use the name `status` for the "status" field of the resource.
+
+#### Stepwise long-running operations
+
+In a stepwise long-running operation, the client makes a request to initiate the operation processing and receives a URL in the response where it can obtain the operation results. The [HTTP specification](https://datatracker.ietf.org/doc/html/rfc7231#section-6.3.3) calls the target of this URL a "status monitor" but we will refer to it as the operation-results.
+
+:white_check_mark: **DO** return a `202-Accepted` status code from the request that initiates a stepwise LRO if the processing of the operation was successfully initiated.
+
+:white_check_mark: **DO** perform as much validation of the initial request as practical and return an error response when appropriate.
+
+:white_check_mark: **DO** return the operation-results URL in the `Operation-Location` response header.
```

Comment:
I'm open to that if others think it is better.  I made it separate because it seems more consistent with the "checklist" style of guidelines we are using throughout.